sqlx-postgres/src/options/pgpass.rs

```diff
@@ -43,7 +43,7 @@ fn load_password_from_file(
 ) -> Option<String> {
     let file = File::open(&path)
         .map_err(|e| {
-            tracing::warn!(
+            tracing::debug!(
```

Comment:
This should only be downgraded to `DEBUG` if the error kind is `NotFound`. If the file exists but we were unable to read it, the user likely wants to know about that.

Reply:
Mh yeah, that's a good point, I'll change it. Do you want testing coverage for that (disclaimer: I currently don't even know if I could test that)

Reply:
No, the change is trivial enough that it shouldn't need a test.